pkg/database/migrations/common.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import (
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+const (
+	SQLiteDialectorName   = "sqlite"
+	PostgresDialectorName = "postgres"
+)
```

Comment:
these constants are now declared in 2 places...

Reply:
also we should get rid of them and use more robust definitions like `sqlite.Dialector{}.Name()` and `postgres.Dialector{}.Name()` — but this is for another PR

Reply:
this was to avoid an import cyclo, so i'll need to find a new package instead of `database` for a shared location

Reply:
then just go for the good definitions I mentioned above — we should probably not have introduced these constants in the first place